### PR TITLE
DreamHost API stub

### PR DESCRIPTION
**Pull request by techtonik**
This allows to list domain for DreamHost API (#56). Current output is a list of records like `{'content': u'64.90.40.29', 'type': u'A', 'name': u'www.ruby.rainforce.org'},`

### CODE REVIEW

**Comment by CapsuleCD** [comment]
Hi.

I'm an automated pull request bot named [CapsuleCD](http://www.github.com/AnalogJ/capsulecd). I handle testing, versioning and package releases for this project. 
- If you're the owner of this repo, you can click the button below to kick off the CapsuleCD build pipeline and create an automated release.'
- If not, don't worry, someone will be by shortly to check on your pull request. 

[CapsuleCD](http://www.capsulecd.com/#!/deploy/github/AnalogJ/lexicon/pullrequests/60)

---
If you're interested in learning more about [CapsuleCD](http://www.github.com/AnalogJ/capsulecd), or adding it to your project, you can check it out [here](http://www.github.com/AnalogJ/capsulecd)

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 requests[security]
-tldextract
\ No newline at end of file
+tldextract
+dreamhostapi
```
Comment (AnalogJ) [inline requirements.txt]: can you move the `dreamhostapi` dependency to the `optional-requirements.txt` file?

**Comment by AnalogJ** [comment]
@techtonik This looks like a good start. I made a comment about the `dreamhostapi` dependency.

**Comment by AnalogJ** [comment]
Hey @techtonik 

Thanks for the PR. I finally got around to putting a contributing.md guide together. It has instructions on how to write a provider, manually test it and run it through the integration suite (with VCRpy recordings).

Can you take a look and update your PR with the VCRpy recordings? Then I can merge your PR immediately :)

**Comment by AnalogJ** [comment]
@techtonik ping :)

**Comment by AnalogJ** [comment]
Closing this, there doesn't seem to be any activity.

**Comment by techtonik** [comment]
https://api.dreamhost.com/ gives 503 today. No luck. =)

**Pull request closed without merging**